### PR TITLE
Add unit suffix to property text-field in SubmodelElementGroup

### PR DESCRIPTION
**Pull request by seicke**
## Description of Changes

Unlike in the [Property view](https://github.com/eclipse-basyx/basyx-applications/blob/main/aas-gui/Frontend/aas-web-gui/src/components/SubmodelElements/Property.vue) (see [NumberType.vue](https://github.com/eclipse-basyx/basyx-applications/blob/main/aas-gui/Frontend/aas-web-gui/src/components/SubmodelElements/ValueTypes/NumberType.vue#L13)), the unit of a property is not displayed in the [SubmodelElementGroup view](https://github.com/eclipse-basyx/basyx-applications/blob/main/aas-gui/Frontend/aas-web-gui/src/components/UIComponents/SubmodelElementGroup.vue#L36C29-L41C44):

<kbd>Bildschirmfoto 2024-06-15 um 11 26 44</kbd>
<kbd>Bildschirmfoto 2024-06-15 um 11 26 54</kbd>

This PR adds the unit suffix for properties in the SubmodelElementGroup.vue:
<kbd>Bildschirmfoto 2024-06-15 um 11 27 27</kbd>

## Related Issue

None

## BaSyx Configuration for Testing

Default configuration of aas-environment, aas-registry, sm-registry, aas-discovery and aas-gui (see [BaSyx Starter Kit](https://basyx.org/#/get-started))

## AAS Files Used for Testing

[TechnicalDataDemo.aasx.zip](https://github.com/user-attachments/files/15846360/TechnicalDataDemo.aasx.zip)

## Additional Information

None

### CODE REVIEW

File: aas-gui/Frontend/aas-web-gui/src/mixins/SubmodelElementHandling.ts
Comment (aaronzi) [inline aas-gui/Frontend/aas-web-gui/src/mixins/SubmodelElementHandling.ts:1]: Just for my understanding, why was this addition necessary?

**Review by aaronzi — APPROVED** [review]

File: aas-gui/Frontend/aas-web-gui/src/mixins/SubmodelElementHandling.ts
Comment (seicke) [reply aas-gui/Frontend/aas-web-gui/src/mixins/SubmodelElementHandling.ts:1]: As example: [SubmodelElementGroupView.vue](https://github.com/eclipse-basyx/basyx-applications/blob/main/aas-gui/Frontend/aas-web-gui/src/components/UIComponents/SubmodelElementGroup.vue) is called by [SubmodelElementCollection.vue](https://github.com/eclipse-basyx/basyx-applications/blob/main/aas-gui/Frontend/aas-web-gui/src/components/SubmodelElements/SubmodelElementCollection.vue) / [SubmodelElementList.vue](https://github.com/eclipse-basyx/basyx-applications/blob/main/aas-gui/Frontend/aas-web-gui/src/components/SubmodelElements/SubmodelElementList.vue) which are called by [SubmodelElementView.vue](https://github.com/eclipse-basyx/basyx-applications/blob/main/aas-gui/Frontend/aas-web-gui/src/components/SubmodelElementView.vue)

Here, the [submodelElementData object](https://github.com/eclipse-basyx/basyx-applications/blob/319d484c01f51266cbd4c8eb00bb2a6045492a8a/aas-gui/Frontend/aas-web-gui/src/components/SubmodelElementView.vue#L136C13-L136C32) contains the ConceptDescription of the SubmodelElement but not the ConceptDescription of its SubmodelElements (in case of modelType = SubmodelElementCollection resp. modelType = SubmodelElementList)

From my point of view one possible solution could also have been a recursive getConceptDescriptions() for all SubmodelElements of a SubmodelElementCollection/SubmodelElementList in [getCD() of SubmodelElementView.vue](https://github.com/eclipse-basyx/basyx-applications/blob/319d484c01f51266cbd4c8eb00bb2a6045492a8a/aas-gui/Frontend/aas-web-gui/src/components/SubmodelElementView.vue#L291), or
just try to the get ConceptDescription of a property if it is missing here in the unitSuffix method of the SubmodelElementHandling mixin.

I decided for the second one, cause of it's easy implementation ;-)

Comment (aaronzi) [reply aas-gui/Frontend/aas-web-gui/src/mixins/SubmodelElementHandling.ts:1]: Makes sense 👍🏻
